### PR TITLE
[Tests] jortestkit submodule

### PR DESCRIPTION
**Pull request by dkijania**
to avoid code duplication between jormugandr and vit testing i introduced jortestkit project. It contians universal tools for testing. For example openssl which would be used for tls testing

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,6 +13,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v1
+      - uses: Mushus/checkout-submodule@v1
```
Comment (danielSanchezQ) [inline .github/workflows/main.yml:16]: I think in the checkout one you can ask for the submodules directly. How do we do it in jormungandr?

Comment (dkijania) [reply .github/workflows/main.yml:16]: will try with actions/checkout@v2

**Comment by dkijania** [comment]
nope, doesn't work

**Comment by danielSanchezQ** [comment]
Use the submodules statement:
```
- uses: actions/checkout@v2
  with:
    submodules: true
```

You can find some help [here](https://github.com/actions/checkout#usage)

**Comment by dkijania** [comment]
thanks! that solves problem

**Pull request merged**